app/build.gradle.kts

```diff
@@ -1,6 +1,9 @@
 plugins {
     alias(libs.plugins.android.application)
     alias(libs.plugins.jetbrains.kotlin.android)
+    id("org.sonarqube") version "5.0.0.4638"
```

Comment:
Whay is the use of this?

Reply:
This is to add a library that helps me get a better report on code quality.